esp-hal/src/timer/systimer.rs

```diff
@@ -145,6 +145,25 @@ impl<'d> SystemTimer<'d> {
         xtal_freq_mhz as u64 * MULTIPLIER
     }
 
+    /// Convert nanoseconds to ticks.
+    // IDF source: uint64_t systimer_ticks_to_us(uint64_t ticks)
+    pub(crate) fn ns_to_ticks(ticks: u64) -> u64 {
```

Comment:
Please call the argument `nanos` if it converts from time to ticks.

But also, this is simply incorrect because it would convert 1 ns to 16 ticks instead of 16 nanos to 1 tick.